### ISSUE TITLE
An internal error occurred while attempting to tab-complete this command

### ISSUE THREAD

**Issue by jshalbrook** [team-abnormals/abnormals-core#102]
When I load abnormals-core-1.16.5-3.1.1.jar, I get an in-game error "An internal error occurred while attempting to tab-complete this command". Without this library everything works fine. This also breaks Essentialsx and PermissionsEx. 

Attached is a pdf of the mod and plugin list and an in-game screen shot of the error. Also attached the log. Any idea how to fix this? We really need this library for five mods that we plan to use on our server.

[latest.log](https://github.com/team-abnormals/abnormals-core/files/6472798/latest.log)
[error1-20210513.pdf](https://github.com/team-abnormals/abnormals-core/files/6472799/error1-20210513.pdf)

**Comment by SmellyModder** [comment]
Abnormals Core doesn't do anything command related.
Can you send the part of the log where the error is logged?

**Comment by jshalbrook** [comment]
That is the odd thing, there is really nothing in the log (which is attached above) that calls out the error (also attached above) that we are seeing in game. Just can't figure this out. All I know with the limited mods we have installed now and simply removing the abnormals jar file and having the error go away is head scratching.

**Comment by jshalbrook** [comment]
I believe this is where everything is going sideways.

[09:26:54] [Server thread/ERROR] [Essentials/]: java.lang.AssertionError: java.lang.NoSuchFieldException: ABNORMALS_CORE_BOAT
[09:26:54] [Server thread/WARN] [/]: java.lang.AssertionError: java.lang.NoSuchFieldException: ABNORMALS_CORE_BOAT

**Comment by jshalbrook** [comment]
We ripped out Essentials and everything is working now. This is really odd, being that we had essentials loaded and removed abnormal and all worked. We loaded all of our mods without abnormal and had the same error as the attached pdf. Ripped out Essentials and started the server back up  with abnormal and low and behold, it is working. @SmellyModder thanks for responding and making us work harder to finding the root cause.
